wallet/manager.go

```diff
@@ -380,7 +382,12 @@ func NewManager(cm ChainManager, store Store, opts ...Option) (*Manager, error)
 			m.mu.Lock()
 			// update the store
 			lastTip, err := store.LastCommittedIndex()
-			if err != nil {
+			if err != nil && strings.Contains(err.Error(), "missing block at index") {
```

Comment:
(nit) expose an error that you can check for rather than checking a string

Reply:
Needs to happen in `coreutils`